### ISSUE TITLE
call stack in bash is hard to look at and overwhelming

### ISSUE THREAD

**Issue by stevekuznetsov** [openshift/origin#6325]
For instance, with a typo for `hack/build-go.sh` I get:
```
$ hack/build-go.sh cmd/k
++ Building go targets for linux/amd64: cmd/k
can't load package: package github.com/openshift/origin/cmd/k: cannot find package "github.com/openshift/origin/cmd/k" in any of:
	/usr/local/go/src/github.com/openshift/origin/cmd/k (from $GOROOT)
	/home/stevekuznetsov/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/cmd/k (from $GOPATH)
	/home/stevekuznetsov/go/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/openshift/origin/cmd/k
!!! Error in hack/../hack/common.sh:144
	'go install "${goflags[@]:+${goflags[@]}}" -ldflags "${version_ldflags}" "${binaries[@]}"' exited with status 1
Call stack:
	1: hack/../hack/common.sh:144 os::build::build_binaries(...)
	2: hack/build-go.sh:15 main(...)
Exiting with status 1
!!! Error in hack/../hack/common.sh:116
	'( os::build::setup_env; local version_ldflags; version_ldflags=$(os::build::ldflags); local goflags; eval "goflags=(${OS_GOFLAGS:-})"; local arg; for arg in "$@";
do
    if [[ "${arg}" == -* ]]; then
        goflags+=("${arg}");
    fi;
done; os::build::export_targets "$@"; local platform; for platform in "${platforms[@]}";
do
    os::build::set_platform_envs "${platform}"; echo "++ Building go targets for ${platform}:" "${targets[@]}"; go install "${goflags[@]:+${goflags[@]}}" -ldflags "${version_ldflags}" "${binaries[@]}"; os::build::unset_platform_envs "${platform}";
done )' exited with status 1
Call stack:
	1: hack/../hack/common.sh:116 os::build::build_binaries(...)
	2: hack/build-go.sh:15 main(...)
Exiting with status 1
```

There's so many lines of poorly formatted output it is difficult for my mind to parse what is going on and what happened. The call stack doesn't actually serve its purpose of being useful here.

@smarterclayton we need to rethink that feature

**Comment by smarterclayton** [comment]
I've got 99 problems but this ain't one :)

**Comment by stevekuznetsov** [comment]
Fixed in #8527
